pypeit/tests/test_runpypeit.py

```diff
@@ -150,5 +159,7 @@ def test_run_pypeit():
 
     # Clean-up
     shutil.rmtree(outdir)
+    '''
     shutil.rmtree(testrawdir)
+    '''
```

Comment:
Here and above, please don't comment out code using the docstring syntax.  Use # (as annoying as it is do this for each line), or just remove the old code.

Reply:
agreed.  removed that code